DateTime.ts

```diff
@@ -15,4 +15,12 @@ export namespace DateTime {
 	export function now(): DateTime {
 		return create(new Date())
 	}
+	export function localize(value: Date, locale?: string, options?: object): DateTime {
```

Comment:
Remove `options`. Be more opinionated and don't leak the underlying implementation, abstract instead.